ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.jsonschema.json

```diff
@@ -1266,6 +1266,11 @@
             "type": "boolean",
             "title": "Shutdown"
           },
+          "load_interval": {
+            "type": "integer",
```

Comment:
Do we want to validate the values?
 "default": 300,
 "minimum": 0,
 "maximum": 600,

Reply:
Min and max yes. No default in eos_cli_config_gen. We will just configure whatever value is given (within the min/max), and don't configure anything if set set. This is required since defaults could be set differently, and defaults might even be set with other tooling like configlets etc.

Reply:
updated with min & max.  omitting default as load-interval 300 would show on the interface output when not required.

Reply:
Seems like you have not rerun the schema. Run `pre-commit run schemas --all`